nextcord/application_command.py

```diff
@@ -694,12 +694,38 @@ def from_callback(
             if option_class:
                 first_arg = True
                 typehints = typing.get_type_hints(self.callback)
+                callback_params = signature(self.callback).parameters
+                possible_options: Set[str] = set()
+
+                for _, param in callback_params.items():
+                    if param.annotation is param.empty or param.annotation is Interaction:
```

Comment:
surely this also skips options that have no type - this is supported as it defaults to `str`

Reply:
I was thinking about that, but no it doesn't support those
There isn't a good way to distinguish between the two without doing name checking (which is something I want to avoid)

Reply:
could it not at least check if its the first parameter when its empty

Reply:
Yeah good point, I'll look into that